src/typescript/templates/todoTemplate.ts

```diff
@@ -20,7 +20,7 @@ class TodoTemplate {
                     <p class="todo-info-text">Category: ${todo.category.name}</p>
                 </div>
                 <form>
-                    <input type="checkbox" />
+                    <input type="checkbox" ${todo.status ? 'checked' : ''} />
```

Comment:
If you use form, it should be have form action (Like submit, cancel). For now, it just only a input. So consider use input without form